contracts/REXMarket.sol

```diff
@@ -827,10 +808,11 @@ abstract contract REXMarket is Ownable, SuperAppBase, Initializable {
         address _agreementClass,
         bytes32, //_agreementId,
         bytes calldata _agreementData,
-        bytes calldata // _ctx
+        bytes calldata _ctx
     ) external view virtual override returns (bytes memory _cbdata) {
         _onlyHost();
-        _onlyExpected(_superToken, _agreementClass);
+        if (!_isInputToken(_superToken) || !_isCFAv1(_agreementClass))
+            return _ctx;
```

Comment:
This stakement only allows call agreements with the CFA, the bug was that `_onlyExpected` let IDAs through, so when someone unsubscribed, it created an issue for the app logic